src/LandingPage.tsx

```diff
@@ -65,31 +67,51 @@ const SignIn = styled.div`
 `;
 
 const LandingPage: React.FC = () => {
+  const auth = useContext(AuthContext);
+
   function loginSuccessHandler(
-    response: GoogleLoginResponse | GoogleLoginResponseOffline
+    googleResponse: GoogleLoginResponse | GoogleLoginResponseOffline
   ) {
-    console.log(response);
-    let onlineResponse = response as GoogleLoginResponse;
+    let onlineResponse = googleResponse as GoogleLoginResponse;
     if (!onlineResponse) {
       // Not supported yet
       return;
     }
-    // TODO: Send onlineResponse.tokenId to the backend to validate the token, get the user object, and store that in localStorage
-    localStorage.setItem("token", onlineResponse.tokenId);
-    localStorage.setItem("user", onlineResponse.profileObj.name);
+
+    const options = {
+      credentials: "include" as const,
+      method: "POST",
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+      body: `idtoken=${onlineResponse.tokenId}`,
+    };
+    return fetch(`http://paralibrary.digital/api/login`, options)
+      .then(async (response) => {
+        if (response.status === 200) {
+          let json = await response.json();
+          auth.login({ authenticated: true, userId: json.userId });
```

Comment:
Sends the response from the backend to the auth context provider so that it can be stored in the cookie